src/App.jsx

```diff
@@ -14,45 +15,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+function sortArray(goods, howSort, reverse) {
+  const array = [...goods];
+
+  array.sort((value1, value2) => {
+    if (howSort === SORT_NAME) {
+      return value1.localeCompare(value2);
+    }
+
+    if (howSort === SORT_LENGTH) {
+      return value1.length - value2.length;
+    }
+
+    return 0;
+  });
+
+  if (reverse) {
+    array.reverse();
+  }
+
+  return array;
+}
+
+const SORT_NAME = 'name';
+const SORT_LENGTH = 'lenght';
+
+export const App = () => {
+  const [sort, setSort] = useState('');
+  const [reverse, setIsReversed] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
The state name and setter function name should be corresponding